OMX-27-firmware/OMX-27-firmware.ino

```diff
@@ -1363,13 +1541,31 @@ void loop() {
 
 					// NOTE SELECT
 					if (noteSelect){
-						if (noteSelection) {		// SET NOTE
-							stepSelect = false;
-							selectedNote = thisKey;
-							int adjnote = notes[thisKey] + (octave * 12);
-							stepNoteP[playingPattern][selectedStep].note = adjnote;
-							if (!playing){
-								seqNoteOn(thisKey, defaultVelocity, playingPattern);
+						if (noteSelection) {		// SET NOTE		
+
+//							stepSelect = false;
+//							selectedNote = thisKey;
+//							int adjnote = notes[thisKey] + (octave * 12);
+//							stepNoteP[playingPattern][selectedStep].note = adjnote;
+//							if (!playing){
+//								seqNoteOn(thisKey, defaultVelocity, playingPattern);
+
+							// left and right keys change the octave
+							if (thisKey == 11 || thisKey == 26) {
```

Comment:
I thought it might work better for note select to keep them at opposite ends - rather than adjacent like in midi mode (which will get a few more aux functions perhaps)